Patch by Tim Pope <debian-bug@rebelongto.us> to obey the boldmode setting.
See Debian bug #347790.

Index: xterm/fontutils.c
===================================================================
--- xterm.orig/fontutils.c	2006-10-23 22:34:24.000000000 +0200
+++ xterm/fontutils.c	2006-11-10 02:30:07.000000000 +0100
@@ -886,6 +886,12 @@
 	fnts[fBold] = fnts[fNorm];
     }
 
+    if (!screen->bold_mode && fnts[fBold] != fnts[fNorm]) {
+        TRACE(("...ignoring bold font, as per boldMode\n"));
+	XFreeFont(screen->display, fnts[fBold]);
+	fnts[fBold] = fnts[fNorm];
+    }
+
     if_OPT_WIDE_CHARS(screen, {
 	if (fnts[fWide] != 0
 	    && fnts[fWBold] != 0
